timeutil/schedule.go

```diff
@@ -152,13 +330,20 @@ var weekdayMap = map[string]int{
 	"sat": 6,
 }
 
+var weekdayOrder = []string{
+	"mon",
```

Comment:
Shouldn't we support days with capital letters too? I haven't followed the discussion about syntax, so forgive me if that has been rejected or something.